omero/sysadmins/version-requirements.txt

```diff
@@ -1750,12 +1743,26 @@ Distribution support
       - N/A
       - N/A
     * - 1.6
-      - N/A
+      - EPEL
       - 21, (22)
       - 14.10, (15.04)
       - (8.0), 7.0 bpo
       - Yes
       - Yes
+    * - 1.8
+      - N/A
+      - N/A
+      - N/A
+      - N/A
+      - Yes
+      - Yes
+    * - 1.9
+      - N/A
+      - N/A
+      - 14.10
```

Comment:
sorry I think I am not getting this table right, 1.9 in general should not be there are it is develop